doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst

```diff
@@ -3,6 +3,9 @@
 LE Audio controller for nRF5340
 ###############################
 
+.. warning::
+   The LE Audio Controller for nRF5340 is deprecated, and will be removed in NCSDK 2.7.0 release.
+
 .. contents::
    :local:
    :depth: 2
```

Comment:
I guess we can remove line 20 or even the entire note in lines 18-20, as the state moved from experimental to deprecated?
```
Make sure to always use the latest version of the controller.
```

Reply:
Good point, I removed the note all together - it just contradicts the warning either way